docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:lts-alpine AS build
```

Comment:
(The "docker" directory)

Reply:
yup, completely slipped my mind. Good catch!

Reply:
Actually, I just realized the project doesn't have an `.npmignore`. Instead the files to include in the package are explicitly specified in the `package.json`:
```json
  "files": [
    "append-to-array.js",
    "markdownlint-cli2.js",
    "markdownlint-cli2-fix.js",
    "merge-options.js",
    "resolve-and-require.js"
  ],
```

the `npm pack` command additionally includes `README.md`, `LICENSE`, and `package.json` automatically. 

Because of this, I don't think the project needs a `.npmignore` file. Have I understood correctly?